From f6dfc87c551b4a988de9fdf2b0fa08433fab62e5 Mon Sep 17 00:00:00 2001
From: Marquise Stein <botocratic@gmail.com>
Date: Sat, 7 Nov 2020 06:18:31 -0800
Subject: [PATCH] Replace stream package with readable-stream

---
 index.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/index.js b/index.js
index 1339ffc..a40ece9 100644
--- a/index.js
+++ b/index.js
@@ -1,6 +1,6 @@
 "use strict";
 
-var stream = require("stream");
+var stream = require("readable-stream");
 
 function DuplexWrapper(options, writable, readable) {
   if (typeof readable === "undefined") {
-- 
2.17.1

