From e1fa50f5330e10b3f9196ef68da067f6bfca2248 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 25 Jul 2020 11:13:30 -0700
Subject: [PATCH] Don't fix readable-stream at version 1.0.x

---
 polyfills/browserify-fs.js | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/polyfills/browserify-fs.js b/polyfills/browserify-fs.js
index 9921240..d1b19d6 100644
--- a/polyfills/browserify-fs.js
+++ b/polyfills/browserify-fs.js
@@ -5185,7 +5185,7 @@ var dependencies = {
 	"deferred-leveldown": "~0.2.0",
 	errno: "~0.1.1",
 	prr: "~0.0.0",
-	"readable-stream": "~1.0.26",
+	"readable-stream": "~3.0.0",
 	semver: "~2.3.1",
 	xtend: "~3.0.0"
 };
@@ -5451,8 +5451,6 @@ var util$1 = {
  * MIT License <https://github.com/rvagg/node-levelup/blob/master/LICENSE.md>
  */
 
-// NOTE: we are fixed to readable-stream@1.0.x for now
-// for pure Streams2 across Node versions
 var Readable$1      = readable.Readable
   , inherits      = util$2.inherits
   , EncodingError = errors.EncodingError
-- 
2.17.1

